README.md

```diff
@@ -248,6 +261,32 @@ the `onLeave` and `onEnter` callback will be called. By using the arguments
 passed to the callbacks, you can determine whether the waypoint has crossed the
 top boundary or the bottom boundary.
 
+## Children
+
+If you don't pass a child into your Waypoint, then you can think of the
+waypoint as a line across the page. Whenever that line crosses a
+[boundary](#offsets-and-boundaries), then the `onEnter` or `onLeave` callbacks
+will be called.
+
+If you do pass a child, it must be a single DOM Element (eg; a `<div>`).
```

Comment:
If a Component isn't passable (they're not, right?), then it might be a good idea to explicitly say that here. i.e.;

> If you do pass a child, it must be a single DOM Element (eg; a `<div>`), and **not** a Component (eg; `<MyCustomComponent>`).